kolibri/content/api.py

```diff
@@ -340,3 +339,19 @@ class FileViewset(viewsets.ReadOnlyModelViewSet):
 
     def get_queryset(self):
         return models.File.objects.all()
+
+
+class ContentNodeFileSizeViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = serializers.ContentNodeGranularSerializer
```

Comment:
Does this break when this is not specified? (As this is not used, might be better just not to specify it, as otherwise it is a red herring).

Reply:
@rtibbles  Do you mean the serializer_class? Yes, it will raise an error `AssertionError: 'ContentNodeFileSizeViewSet' should either include a `serializer_class` attribute, or override the `get_serializer_class()` method.` if the serializer_class is not specified.